ingestion_tools/docs/enqueue_runs.md

```diff
@@ -153,6 +153,36 @@ python3 enqueue_runs.py sync --exclude '*' --include 'Annotations/*.json' --s3-p
 | --include-deposition | null    | Look for deposition metadata  with the deposition ids passed here. This helps sync the deposition data.                                                                                            |
 | --no-sync-dataset | False   | Skip syncing datasets. This is useful when we want to only update deposition data                                                                                                                  |
 
+## S3 File validation (`validate` subcommand)
+
+The `enqueue_runs.py validate` command queues up jobs to validate dataset data in s3. It *requires* one or more dataset ID's to be specified. Once the jobs are complete, you can view test results for staging or prod by updating the following URLs with the necessary dataset ID's:
+
+https://files.cryoet.staging.si.czi.technology/prod_validation/10168/index.html
+https://files.cryoet.staging.si.czi.technology/staging_validation/10168/index.html
```

Comment:
This is awesome, but I'm getting an Access Denied message for the staging link. Prod link works.

Reply:
<img width="807" alt="image" src="https://github.com/user-attachments/assets/725a525b-b6b3-43ba-8725-e390c0f70e2a" />

Reply:
Yep, I haven't run the validation on every dataset .... YET

Reply:
Sorry, no rush, I didn't realize that was the reason 😅